### PR TITLE
"Events and observers" page: Added warning about overriding observer by using existing name.

### PR DESCRIPTION
**Pull request by vasilii-b**
## Purpose of this pull request

This pull request (PR) adds a warning message about the observer's unique name to the "Events and observers" page 

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/events-and-observers.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3.4/lib/internal/Magento/Framework/Event/etc/events.xsd#L16

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/extension-dev-guide/events-and-observers.md [hunk 1]
```diff
@@ -127,6 +127,9 @@ The `observer` [xml](https://glossary.magento.com/xml) element has the following
 *  `disabled` - Determines whether this observer is active or not. Default value is false.
 *  `shared` - Determines the [lifestyle]({{ page.baseurl }}/extension-dev-guide/build/di-xml-file.html#object-lifestyle-configuration) of the class. Default is `true`.
 
+{: .bs-callout .bs-callout-warning}
+The observer `name` must be unique. Otherwise, an override happen.
```
Comment (shrielenee) [inline src/guides/v2.3/extension-dev-guide/events-and-observers.md:131]: Rephrase to be clearer. Maybe "The observer `name` must be unique, or an override may occur." or similar.

**Comment by vasilii-b** [comment]
Hi @shrielenee,
Thanks for the suggestion. Could you please check the adjusted text?
Thank you!

**Comment by shrielenee** [comment]
running tests

**Pull request merged**